backend/src/main/java/com/senior/project/backend/domain/Student.java

```diff
@@ -3,18 +3,43 @@
 import lombok.*;
 
 import java.util.Date;
+import java.util.UUID;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+
+@Entity
 @Data
 @ToString
 @NoArgsConstructor
 @AllArgsConstructor
 @Generated
-public class Student {
+public class Student extends User{
 	
-    private String id;
+	@Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID studentId;
 	
-	private int userID;
 	private int universityID;
+	private double gpa;
+	private String description;
+	private String degreeLevel;
```

Comment:
is this supposed to be year level?

Reply:
I'm going based off of what was said on the pages but yes.

Reply:
what pages?